src/stabilization_guide.md

```diff
@@ -96,6 +99,24 @@ require steps beyond what this guide talks about.
 Note: Before we stabilize any feature, it's the rule that it
 should appear in the documentation.
 
+### Determining the stabilization version
+
+The version in which the feature will be stabilized *must* match
+the value of [the `src/version` file in `master`] when the PR is merged.
+
+It's worth checking [the version schedule on the Forge] to see whether
+changes are coming soon.  You'll usually use the version labelled "Nightly".
+That's two versions higher than the current stable release,
+as what's currently in beta will be the next stable release,
```

Comment:
```suggestion
"Nightly" is two versions higher than the current stable release,
since what's currently in beta will be the next stable release,
```